### ISSUE TITLE
No technical discussion of level set

### ISSUE THREAD

**Issue by drjfloyd** [firemodels/fds#11681]
The Technical Reference Guide does not discuss the equations and assumptions of the FDS implementation of the level set method.

**Comment by rmcdermo** [comment]
This issue was prompted by [this discussion thread](https://mail.google.com/mail/u/0/?tab=rm&ogbl#inbox/FMfcgzGsmDkTrXRGKkddvhTbgKswRLbf).  Would be good to make sure these questions are addressed in the guide.

**Comment by mcgratta** [comment]
I don't know if the level set method as coded up in FDS now is the same as when I coded it. In any event, we should document whatever we are using.

**Comment by ericvmueller** [comment]
My assessment is that the technical approach, which I assume was coded up by you, follows that presented in [the IJWF paper](https://research.fs.usda.gov/treesearch/59231). I've made some minor tweaks to fix some bugs and expand functionality for custom fuels, but nothing to the fundamental method. But I can certainly work on pulling over the details from that paper into the guide and making sure it all agrees.

**Comment by ericvmueller** [comment]
Yeah I can definitely appreciate that. I think it could be a worthwhile exercise though ...and perhaps timely because I was thinking about introducing an option related to one of these assumptions (the re-scaling wind speed from aloft) which is problematic for a case I have. We can discuss more next week.

**Comment by ericvmueller** [comment]
@mcgratta I pushed up a first version of a chapter on the level set for the tech guide with #14740

I'm sure there's more that can be added (for example a figure of the elliptical model similar to Tony's paper might be useful). And there may also need to be some tweaks for consistency with general FDS nomenclature, etc. So please give it a read when you get a chance ...and feel free to edit as you see fit.

**Comment by mcgratta** [comment]
I added some space to the equations to make them easier to read. Also, note that LaTeX uses forward and backward quotes. 
Otherwise, this is a handy write-up. Thanks.

**Comment by ericvmueller** [comment]
Thanks for the edits - going to close this with the understanding that it will get another once-over during the documentation review for v7.
